src/compiler/program.ts

```diff
@@ -1448,7 +1446,7 @@ namespace ts {
 
             // file.imports may not be undefined if there exists dynamic import
             let imports: StringLiteral[];
-            let moduleAugmentations: StringLiteral[];
+            let moduleAugmentations: Array<StringLiteral | Identifier>;

```

Comment:
For my own edification, what's the difference between `T[]` and `Array<T>`?

Reply:
No semantic difference. I prefer to use `Array<>` when wrapping unions to avoid needing parentheses.